### DISCUSSION TITLE
How to use walletconnect + viem?

### DISCUSSION THREAD

**Discussion by alexbakers** [wevm/viem#268] [Question] [answered]
Could you give me an example?

**Comment by jxom** [comment, marked as answer]
Here you go! 

https://stackblitz.com/edit/viem-getting-started-a3ekcn?file=index.tsx

**Reply by siosio34** [reply]
not working ...

**Reply by tmm** [reply]
What specifically is not working? Works fine for me. Make sure you set a `projectId`.

**Reply by naz3eh** [reply]
Hey @jxom! I am trying to use the `WalletConnect` with `viem` using the configuration you have mentioned above. But I am getting this error. 
<img width="391" alt="image" src="https://github.com/wagmi-dev/viem/assets/56908732/3ecef760-9680-477a-aa5f-81add3cebc08">


Here is the [code ref](https://github.com/Nazeeh21/comp-kit/blob/0184fd3994f0f16f5e6db31125879feb3e7b5ed3/package/src/components/Wallets/wallectConnectWallet.tsx#L14) for more context. Can you please help me out with this?
